docs/source/index.rst

```diff
@@ -8,13 +8,17 @@ SkyPilot Documentation
 
 .. raw:: html
 
-    <script async defer src="https://buttons.github.io/buttons.js"></script>
     <p style="text-align:center">
-    <a class="github-button" href="https://github.com/skypilot-org/skypilot" data-size="large" data-show-count="true" aria-label="Star skypilot-org/skypilot on GitHub">Star</a>
+    <a class="reference external image-reference" style="vertical-align:10px" href="https://skypilot.readthedocs.io/en/latest/index.html"><img alt="https://readthedocs.org/projects/skypilot/badge/?version=latest" src="https://readthedocs.org/projects/skypilot/badge/?version=latest" style="height:27px"></a>
```

Comment:
minor nit: Do we need the docs badge on the docs page? :)

Reply:
Good call! It was following the [ray](https://docs.ray.io/en/latest/index.html)'s doc. Just removed the doc badge. ;)

Reply:
The latest doc (122629e):
<img width="659" alt="image" src="https://user-images.githubusercontent.com/6753189/196622676-8ec1fe7b-9e56-4813-9171-f9f6b75529b5.png">